Jenkinsfile

```diff
@@ -32,10 +32,10 @@ pipeline {
                     }
                 }
 
-                stage('ubuntu-18.04/gcc-7.3.0 (Debug/format/lint/censored)') {
+                stage('ubuntu-20.04/gcc-9.3 (Debug/format/lint/censored)') {
```

Comment:
Good.  I like this convention of major.minor better as it's less likely to be inconsistent with what the package manager installs throughout the 20.04 support cycle.